viewing_party/party.py

```diff
@@ -0,0 +1,201 @@
+def create_movie(movie_title, genre, rating):
+    '''
+    function builds a dictionary with three key value 
+    pairs where each value is the relevant parameter for each movie
+    '''
+    if movie_title == None or genre == None or rating == None:
+        return None
```

Comment:
this works fine! one thing you could consider is maybe checking for falsey rather than _only_ None

```suggestion
    if not movie_title  or not genre or not rating:
        return None
```